src/directive/translate.js

```diff
@@ -5,7 +5,7 @@ angular.module('pascalprecht.translate')
  * @requires $compile
  * @requires $filter
  * @requires $interpolate
- * @restrict A
```

Comment:
Well actually I just saw that it was wrong (as I created a new directive), so I fixed it... Any reason not to?

I know from my experience that it allows elements, and you can see that from the [code](https://github.com/angular-translate/angular-translate/blob/master/src/directive/translate.js#L107).